schemas/rule.yml

```diff
@@ -0,0 +1,83 @@
+---
+- name: rule
+  title: Rule
+  group: 2
+  short: Fields to capture details about rules used by observers and agents to generate alerts or other events.
+  description: >
+   Rule fields are used to capture the specifics of any observer or agent rules that generate alerts or other events..
+   Examples of data sources that would populate the rule fields include: platforms, network or host IDS/IPS, network 
+   firewalls, web access firewalls, url filters, endpoint detection and response (EDR) systems, etc.  
+  
+  type: group
+  fields:
+
+    - name: id
+      level: extended
+      type: keyword
+      short: Rule ID
+      description: >
+        A rule ID that is unique within the scope of an agent, observer, or other entity using the rule for detection of this event
+
+      example: 101 
+
+    - name: uuid
+      level: extended
+      type: keyword
+      short: Rule UUID
+      description: >
+        A rule ID that is unique within the scope of a set or group of agents, observers, or other entities using the rule for detection of this event.
+
+      example: 1100110011
+
+    - name: version
+      level: extended
+      type: keyword
+      short: Rule Version
+      description: The version / revision of the rule being used for analysis.
+      example: 1.1
+
+    - name: name
+      level: extended
+      type: keyword
+      short: Rule Name
+      description: The name of the rule generating the event.
+      example: Filter_DOH
+
+
+    - name: description
+      level: extended
+      type: keyword
+      short: Rule Description
+      description: The description of the rule generating the event.
+      example: Block requests to public DNS over HTTPS / TLS protocols
+
+    - name: type
+      level: extended
+      type: keyword
+      short: Rule Type
+      description: >
+         A categorization value keyword used by the entity using the rule for detection of this event
+
+      example: DNS
```

Comment:
If it's a category, should we call this field `rule.category`?

Reply:
Was thinking of this as more a categorization of what type of action/information is being analyzed by the alert - e.g. url for url filtering, dns for nxdomain results, acl for basic access control, etc.

Reply:
brought this back re: ruleset discussion below, as rule.category